src/function_data.jl

```diff
@@ -64,24 +64,49 @@ representation of cost functions where the points store quantities (x, y), such
 The curve starts at the first point given, not the origin.
 
 # Arguments
- - `points::Vector{Tuple{Float64, Float64}}`: the points (x, y) that define the function
+ - `points::Vector{@NamedTuple{x::Float64, y::Float64}}`: the points that define the function
 """
 struct PiecewiseLinearPointData <: FunctionData
-    points::Vector{Tuple{Float64, Float64}}
+    points::Vector{@NamedTuple{x::Float64, y::Float64}}
```

Comment:
we should create and Alias for this similarly to how it is done in PSY with MinMax adding something like  

```
const TWO_DIM_COORDINATES = NamedTuple.... 
```